PartymodeAutostart.py

```diff
@@ -12,11 +12,8 @@
 
 
 def log(txt):
-    if isinstance(txt, str):
-        txt = txt.decode("utf-8")
```

Comment:
Why have you removed the utf-8 encode/decode?

Reply:
old code doesn't work in python3. see https://www.pythoncentral.io/encoding-and-decoding-strings-in-python-3-x/

Reply:
Great!